runtime/common/src/lib.rs

```diff
@@ -282,9 +284,78 @@ parameter_types! {
 	pub const MaxVestingSchedules: u32 = 100;
 }
 
+// pallet xyk
+parameter_types! {
+	pub const MinTradingLimit: Balance = MIN_TRADING_LIMIT;
+	pub const MinPoolLiquidity: Balance = MIN_POOL_LIQUIDITY;
+	pub const MaxInRatio: u128 = MAX_IN_RATIO;
+	pub const MaxOutRatio: u128 = MAX_OUT_RATIO;
+	pub const RegistryStrLimit: u32 = 32;
+}
+
+parameter_types! {
+	pub const MaxAuthorities: u32 = 32;
+}
+
+parameter_types! {
+	pub const MaxKeys: u32 = 10_000;
+    pub const MaxPeerInHeartbeats: u32 = 10_000;
+    pub const MaxPeerDataEncodingSize: u32 = 1_000;
+}
+
 parameter_types! {
 	pub const SessionDuration: BlockNumber = EPOCH_DURATION_IN_SLOTS as _;
 	pub const ImOnlineUnsignedPriority: TransactionPriority = TransactionPriority::max_value();
 	/// We prioritize im-online heartbeats over election solution submission.
 	pub const StakingUnsignedPriority: TransactionPriority = TransactionPriority::max_value() / 2;
 }
+
+pub fn get_all_module_accounts() -> Vec<AccountId> {
+    vec![
+        TreasuryPalletId::get().into_account(),
+    ]
+}
+
+pub struct DustRemovalWhitelist;
+
+impl Contains<AccountId> for DustRemovalWhitelist {
+    fn contains(a: &AccountId) -> bool {
+        get_all_module_accounts().contains(a)
+    }
+}
+
+use hydradx_traits::CanCreatePool;
+pub struct AllowAnyPool;
+
+impl CanCreatePool<AssetId> for AllowAnyPool {
+	fn can_create(_asset_a: AssetId, _asset_b: AssetId) -> bool {
+		true
+	}
+}
+
+
+use pallet_staking::BenchmarkingConfig;
+
+pub struct StakingBenchmarkingConfig;
+
+impl BenchmarkingConfig for StakingBenchmarkingConfig{
+	type MaxValidators = ConstU32<100>;
```

Comment:
is this accurate for our case? we should have like 165 validators with 64 nominators each. anyway it requires rebench probably

Reply:
Set 165 and 64.